src/modules/ekf2/ekf2_main.cpp

```diff
@@ -858,11 +858,14 @@ void Ekf2::run()
 			ev_data.quat = q;
 
 			// position measurement error from parameters. TODO : use covariances from topic
-			ev_data.posErr = _ev_pos_noise.get();
+			ev_data.posErr = fmaxf(_ev_pos_noise.get(), fmaxf(ev_pos.eph, ev_pos.epv));
```

Comment:
Longer term  I will update the ecl EKF to handle separate horiz and vertical accuracy values for ev data.